### PR TITLE
LPS-151201 Node Label is being replaced by node ID in warning alerts at Kaleo Designer

### PR DESCRIPTION
**Pull request by daiazius**

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:relevant

**Review by naraandrade — CHANGES_REQUESTED** [review]
@daiazius There are integration tests failing, please fix them.

**Comment by naraandrade** [comment]
Hey @daiazius, could you please take a look at this pull request?

File: modules/apps/portal-workflow/portal-workflow-kaleo-definition-api/src/main/java/com/liferay/portal/workflow/kaleo/definition/Node.java [hunk 1]
```diff
@@ -89,6 +90,12 @@ public Map<Locale, String> getLabelMap() {
 		return _labelMap;
 	}
 
+	public String getLabelName() {
```
Comment (seltong) [inline modules/apps/portal-workflow/portal-workflow-kaleo-definition-api/src/main/java/com/liferay/portal/workflow/kaleo/definition/Node.java:93]: HI @daiazius , could you please change the method name to `getDefaultLabel()`?

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:sf

**Comment by seltong** [comment]
Hi @daiazius could you please run the `gw baseline` command in this directory `modules/apps/portal-workflow/portal-workflow-kaleo-definition-api/`?
It will create some files. You can add this files in a commit with the message: `LPS-151201 Semver`.
If you have any question, slack me :).

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by seltong** [comment]
Hi @daiazius could you please change the message of this [commit](https://github.com/liferay-immersion/liferay-portal/pull/37/commits/296caf6c07ed0c6e55ce8121897779b6c2017256) to `Add getDefaultLabel method and apply it`?

**Review by seltong — APPROVED** [review]

**Comment by seltong** [comment]
ci:forward

**Comment by seltong** [comment]
nice, @daiazius thx :)

**Comment by seltong** [comment]
ci:forward

**Pull request closed without merging**